pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -200,6 +203,9 @@ public void received(Consumer consumer, Message msg) {
                     limiter.acquire();
                 }
 
+                long latencyMicros = NANOSECONDS.toMicros(System.nanoTime() - msg.getPublishTime());
+                recorder.recordValue(latencyMicros);
```

Comment:
Since the precision is in millis, we could just record it in millis rather than micros.

Reply:
got it, thanks

Reply:
@merlimat Is the PerformanceProducer recored the wrong time? record use millions while the passed value is micros(https://github.com/apache/incubator-pulsar/blob/627885563673e145b850d2160557afdc434ae2fc/pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceProducer.java#L352-L354)